scenario.py

```diff
@@ -60,11 +60,19 @@ def __init__(self, params, experiment_path):
 
         self.federated_test_score = int
 
+        # Define how federated learning aggregation steps are weighted. Toggle between 'uniform' and 'data-volume'
+        # Default is 'uniform'
+        if "aggregation_weighting" in params:
+            self.aggregation_weighting = params["aggregation_weighting"]
+        else:
+            self.aggregation_weighting = "uniform"
+
         self.node_list = []
 
         self.contributivity_list = []
 
         self.epoch_count = 40
+        self.minibatch_count = 20
```

Comment:
I would also rename to `minibatch_size `(or even to `federated_learning_batch_size`) as the minibatch_size would generally be greater than the batch_size !

Reply:
Hmm, this I am not a fan. This variable is not for a "size", it is for a "count".